packages/knip/test/cli/cli-reporter-githubActions.test.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { resolve } from '../../src/util/path.js';
+import { exec } from '../helpers/exec.js';
+
+const cwd = resolve('fixtures/module-resolution-non-std');
+
+test('knip --reporter githubActions (files, unlisted & unresolved)', () => {
```

Comment:
I guess it would be great to add this newly created reporter to knip's testing suite.